controller/array_action/array_mediator_svc.py

```diff
@@ -453,13 +454,13 @@ def get_object_by_id(self, object_id, object_type):
             return self._generate_snapshot_response_with_verification(cli_object)
         return self._generate_volume_response(cli_object)
 
-    def _create_similar_volume(self, source_volume_name, target_volume_name):
+    def _create_similar_volume(self, source_cli_volume, target_volume_name, pool):
```

Comment:
please choose a parameters order as you see fit and ensure it's aligned with `_create_snapshot`